pyproject.toml

```diff
@@ -147,7 +147,7 @@ test = { cmd = "pytest -vsl --cov=skrub --cov-report=xml skrub" }
 
 [tool.pixi.environments]
 lint = ["lint"]
-doc = ["doc"]
+doc = ["optional", "doc"]
```

Comment:
@jeromedockes This should be the culprit since the doc CI only run on changed example.

I trigger a full build here to make sure.